cocodump/base_types.py

```diff
@@ -0,0 +1,168 @@
+from colorama import Fore, Style
+
+from cocodump.colorizer import colorize, colorize_argument
+from cocodump.label_generator import get_label
+
+
+class Instruction:
+    inst: str
+    args: list[str]
+    addr: int
+    inst_bytes: bytearray
+    labels: list[str]
+    comment: str | None
+
+    TEXT_ALIGN = 20
```

Comment:
It looks like this class will benefit significantly from the addition of `__slots__`.

Reply:
Yep.